### PR TITLE
Move page title to Header 

### LINKED ISSUE #864: Move page title to Header
telegram-cloud-photo-size-2-5307594248390042199-y
Move "Problems" to Header
Снимок экрана 2024-08-08 в 19 36 00

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Bug Fix
-   [x] Refactor

## Related issues

Resolve #864

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
U 2

File: src/components/PageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -66,6 +71,9 @@ export const PageHeader = ({ userSettings }: PageHeaderProps) => {
 
     return (
         <div className={s.PageHeader}>
+            <PageTitle title={pageTitle} backlink={backlink}>
+                {titleMenuItems && titleMenuItems.length > 0 && <TitleMenu items={titleMenuItems} />}
```
Comment (asabotovich) [inline src/components/PageHeader/PageHeader.tsx:75]: nullable

File: src/components/PageHeader/PageHeader.tsx [hunk 2]
```diff
@@ -18,9 +20,12 @@ const StyledDescription = styled.div`
 
 interface PageHeaderProps {
     userSettings?: UserSettings;
+    pageTitle: string;
+    backlink?: string;
+    titleMenuItems?: DropdownMenuItem[];
```
Comment (asabotovich) [inline src/components/PageHeader/PageHeader.tsx:25]: just children

**Comment by Katrin-kudryash** [comment]
wrwrwr
qsw
U1

**Review by asabotovich — CHANGES_REQUESTED** [review]

File: src/components/PageHeader/PageHeader.tsx [hunk 3]
```diff
@@ -66,6 +64,9 @@ export const PageHeader = ({ userSettings }: PageHeaderProps) => {
 
     return (
         <div className={s.PageHeader}>
+            <PageTitle title={pageTitle} backlink={backlink}>
```
[the remark this answers is not in the document]
Comment (Katrin-kudryash) [reply src/components/PageHeader/PageHeader.tsx]: done

**Review by asabotovich — APPROVED** [review]

**Review by nyet-ty — APPROVED** [review]

**Pull request merged**